### ISSUE TITLE
Text horizontally out of the viewport does not become visible when vieport size is increased

### ISSUE THREAD

**Issue by synth3** [FXMisc/RichTextFX#412]
I have found the a problem regarding scrolling end re-sizing. Since it's not easy to describe I just post the steps needed to reproduce and a GIF showing the behavior:
- open the RichText demo
- disable "wrap"
- insert one long line of text
- decrease the horizontal size of the window until a horizontal scroll bar is visible
- scroll all the way to the right
- increase the window size until the horizontal scrollbar dissapears
-- Result: The content stays scrolled to the right
-- Expected: Same behavior as for the vertical dimension (which works just fine)

![screenrecording](https://cloud.githubusercontent.com/assets/19573241/21648234/891a267e-d29d-11e6-8ab6-ce08cb7ad9c0.gif)

Sidenote: As horizontal scrolling is broken with Flowless 0.5.1 I used a snapsot that was created from the current state of the Flowless repo.

**Comment by JordanMartinez** [comment]
Good catch. Unfortunately, I don't understand Flowless' inner code that well, so I'm not the one who can determine where the bug is occurring and then fix it.

**Comment by JordanMartinez** [comment]
This is another issue I want to look into after better documenting Flowless' code.

**Comment by JordanMartinez** [comment]
I fixed the bug in TomasMikula/Flowless#42. Just waiting for Tomas to merge it.

**Comment by JordanMartinez** [comment]
I've created a branch in my repo of Flowless called `jitpack-master` that includes the fix I've submitted. If you change your build to use that via Jitpack, then this bug will no longer appear. Follow these instructions to update your build to use it. You can revert your build to the next release whenever Tomas makes the next one including the fix.

### Maven

Add the repository

````xml
<repositories>
    <repository>
        <id>jitpack.io</id>
        <url>https://jitpack.io</url>
    </repository>
</repositories>
````

Add the dependency

````xml
<dependency>
    <groupId>com.github.JordanMartinez</groupId>
    <artifactId>RichTextFX</artifactId>
    <version>jitpack-master-SNAPSHOT</version>
</dependency>
````
### Gradle

````groovy
repositories {
    maven {
        url 'https://jitpack.io'
    }
}

dependencies {
    compile "com.github.JordanMartinez:RichTextFX:jitpack-master-SNAPSHOT"
}
````

### Sbt

````scala
resolvers += "jitpack" at "https://jitpack.io"

libraryDependencies += "com.github.JordanMartinez" % "RichTextFX" % "jitpack-master-SNAPSHOT"
````

**Comment by JordanMartinez** [comment]
This was fixed in the latest snapshot of Flowless (6.0.0-SNAPSHOT), so all one needs to do is refresh their dependencies.
